pkg/controller/machinehealthcheck/machinehealthcheck_controller_test.go

```diff
@@ -364,11 +364,28 @@ func TestReconcile(t *testing.T) {
 	}
 }
 
-func TestHasMachineSetOwner(t *testing.T) {
+func TestHasControllerOwner(t *testing.T) {
+	true := true
```

Comment:
Go let's you redefine a built-in? 🤔 
```suggestion
	truth := true
```

Reply:
`pointer.BoolPtr(true)`